draft-ietf-scitt-architecture.md

```diff
@@ -452,6 +459,36 @@ This specification prioritizes conformance to {{RFC9052}} and its required and o
 Profiles and implementation specific choices should be used to determine admissability of conforming messages.
 This specification is left intentionally open to allow implementations to make the restrictions that make the most sense for their operational use cases.
 
+There are many types of Statements (such as SBOMs, malware scans, audit reports, policy definitions) that Issuers may want to turn into Signed Statements.
```

Comment:
a reviewers note that this is not a new section, but rather a re-ordering. See line [538](https://github.com/ietf-wg-scitt/draft-ietf-scitt-architecture/pull/219/files#diff-585fb424519b79cf000445f3425fa56d328cbaca5d2a622740ddc4e5ca91dbe1L538) for reference